packages/docusaurus-plugin-search/theme/wrapRoot.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React, {useState} from 'react';
+import {FuseContext} from '@theme/useSearch';
+
+const Wrapper = (props) => {
+  const [fuse, setFuse] = useState(undefined);
+  return <FuseContext.Provider value={{fuse, setFuse}} {...props} />;
+};
+
+export default (element) => <Wrapper>{element} </Wrapper>;
```

Comment:
I'm not sure this should be a theme comp, but moved somewhere else. We don't want the user to be able to swizzle such component imho, it's internal code of the plugin, and should not conflict with another plugin deciding to use the exact same filename